### ISSUE TITLE
setCop() and setMaximumTier() does not work together

### ISSUE THREAD

**Issue by brachy84** [codetaylor/artisan-worktables#235]
#### Issue Description

Script with crafttweaker. setMaximumTier simple does not work with setCopy()

#### What Happens

The recipe can be made in all three tiers

#### What You Expect to Happen

The recipe should only be in tier 0

#### Script
https://pastebin.com/7gXRLsuv

#### Affected Versions

* Minecraft: 1.12.2
* Forge: 2847
* CraftTweaker: 1.12-4.1.20.573
* Artisan Worktables: 1.23.0
* Artisan Automation: 1.0.3
* Artisan Integrations: 1.2.1
* Athenaeum: 1.18.0

**Comment by codetaylor** [comment]
Thanks for the report!

I've been able to reproduce this with the following script:

```zs
import mods.artisanworktables.builder.RecipeBuilder;
import mods.artisanworktables.builder.Copy;

// Recipe #1
RecipeBuilder.get("basic")
    .setCopy(
        Copy.byOutput([<minecraft:crafting_table>])
    )
    .setMaximumTier(0)
    .create();

// Recipe #2
RecipeBuilder.get("basic")
  .setShaped([
    [<minecraft:cobblestone>, <minecraft:cobblestone>, <minecraft:cobblestone>],
    [<minecraft:cobblestone>, null, <minecraft:cobblestone>],
    [<minecraft:cobblestone>, <minecraft:cobblestone>, <minecraft:cobblestone>]])
  .setMaximumTier(0)
  .addOutput(<minecraft:furnace>)
  .create();
```

Confirmed that `.setMaximumTier(0)` works as expected with recipe 2, but does not enforce a maximum tier in recipe 1.

**Comment by codetaylor** [comment]
Should be solved in `1.23.2`
